Add CONTRIBUTING.md

gabrielmocanu opened the pull request:
Add CONTRIBUTING.md.
Before this PR is integrated, the discussion tab in this repo should be activated. I don't have enough rights to do that.
@razvand , can you do that?

Signed-off-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

razvand commented:
@gabrielmocanu , I added the `Discussions` tab.

razvand reviewed (approved):
Add commit description to pacify `checkpatch.`

gabrielmocanu commented:
@razvand  Done. 
There is another warning from the checkpatch:
WARNING: added, moved or deleted file (s), does MAINTAINERS need updating?
But I think it's ok.
